mappings/net/minecraft/world/biome/source/util/VanillaBiomeParameters.mapping

```diff
@@ -77,6 +77,11 @@ CLASS net/minecraft/class_6461 net/minecraft/world/biome/source/util/VanillaBiom
 		ARG 1 parameters
 	METHOD method_37718 writeRiverBiomes (Lcom/google/common/collect/ImmutableList$Builder;Lnet/minecraft/class_6452$class_6454;)V
 		ARG 1 parameters
+	METHOD method_37846 getShoreOrNearInlandBiome (IILnet/minecraft/class_6452$class_6454;Lnet/minecraft/class_5321;)Lnet/minecraft/class_5321;
```

Comment:
This one really checks special cases and uses shattered savanna than the fallback in those cases.

Reply:
I think I always named it something like `getBiomeOrShatteredSavanna` in my PRs.

Reply:
Yeah, out of all its usages, one of it is for picking a biome for shore and near-inland; a few others are near-inland, and one usage is like make a call and never made use of the result biome